web/app/js/components/ConfigureProfilesMsg.jsx

```diff
@@ -53,21 +62,39 @@ class ConfigureProfilesMsg extends React.Component {
   };
 
   handleClose = () => {
-    this.setState({ open: false });
+    let state = this.state;
+    state.error['namespace'] = false;
+    state.error['service'] = false;
+    state.open = false;
+    this.setState(state);
   };
 
   handleChange = name => {
     let state = this.state;
 
     return e => {
       state.query[name] = e.target.value;
+      state.error[name] = false;
       this.setState(state);
     };
-  }
+  };
+
+  validateFields = (type, name) => {
+    let state = this.state;
+    let regExp = /[a-z0-9]([-a-z0-9]*[a-z0-9])?/;
```

Comment:
this regexp allows me to enter "a##" and it passes validation. If we look at `IsDNS1035Label` (for service name) and `IsDNS1123Label` (for ns) which are used for validation in the CLI profile command, we see that the full regexp has start and end checks:
```
const dns1035LabelFmt string = "[a-z]([-a-z0-9]*[a-z0-9])?"
const dns1035LabelErrMsg string = "a DNS-1035 label must consist of lower case alphanumeric characters or '-', start with an alphabetic character, and end with an alphanumeric character"
const DNS1035LabelMaxLength int = 63

var dns1035LabelRegexp = regexp.MustCompile("^" + dns1035LabelFmt + "$")
```
So the following should work:
```    
let serviceNameRegexp = RegExp('^[a-z]([-a-z0-9]*[a-z0-9])?$');
let match = serviceNameRegexp.test(name);
```

Note that you'll need a slightly different regexp for namespaces:
```
let namespaceRegexp = RegExp('^[a-z0-9]([-a-z0-9]*[a-z0-9])?$');
```

I also think we should place the regexps in consts and put them either in Utils.js or at the top of this file!